test-cases/microbenchmarking/amazon_perf_simple_query_c7g.yaml

```diff
@@ -9,7 +9,7 @@ use_mgmt: false
 
 instance_type_db: 'c7g.large'
 data_volume_disk_num: 1
-user_prefix: 'perf-microbenchmarking-amazon'
+user_prefix: 'perf-microbenchmarking-amazon-c7g-large'
```

Comment:
I don't think we should have this in the name, no one understands it. just ARM and x86 in the name.

Reply:
renamed